src/device/device-management.ts

```diff
@@ -1,66 +1,65 @@
 import * as media from '../media';
+import { LocalAudioTrack, MicrophoneConstraints } from '../media/local-audio-track';
 import { LocalCameraTrack } from '../media/local-camera-track';
 import { LocalDisplayTrack } from '../media/local-display-track';
 import { LocalMicrophoneTrack } from '../media/local-microphone-track';
+import { CameraConstraints, DisplayConstraints } from '../media/local-video-track';
 
-export enum ErrorTypes {
-  DEVICE_PERMISSION_DENIED = 'DEVICE_PERMISSION_DENIED',
-  CREATE_CAMERA_TRACK_FAILED = 'CREATE_CAMERA_TRACK_FAILED',
-  CREATE_MICROPHONE_TRACK_FAILED = 'CREATE_MICROPHONE_TRACK_FAILED',
-}
+import { ErrorTypes, WebrtcError } from '../error/WebrtcError';
 
 /**
- * Represents a WCME error, which contains error type and error message.
+ * Creates a camera video track. Please note that the constraint params in second getUserMedia call would NOT take effect when:
+ *
+ * 1. Previous captured video track from the same device is not stopped .
+ * 2. Previous createCameraTrack() call for the same device is in progress.
+ * Creates MicrophoneTrack and CameraTrack at the same time.
+ * Creates LocalMicrophoneTrack and LocalCameraTrack at the same time.
```

Comment:
this line is duplicate with line 15, line 15 should be removed

Reply:
removed